qui/updater/intro_page.py

```diff
@@ -175,6 +175,47 @@ def select_rows(self):
             row.selected = row.updates_available \
                            in self.head_checkbox.allowed
 
+    def select_rows_ignoring_conditions(self, cliargs):
+        cmd = ['qubes-vm-update', '--dry-run']
+
+        args = [a for a in dir(cliargs) if not a.startswith("_")]
+        for arg in args:
+            if arg in ("dom0", "no-restart", "restart", "max_concurrency",
+                       "log"):
+                continue
+            value = getattr(cliargs, arg)
+            if value:
+                if arg in ("skip", "targets"):
+                    vms = set(value.split(","))
+                    vms_without_dom0 = vms.difference({"dom0"})
+                    if not vms_without_dom0:
+                        continue
+                    value = ",".join(vms_without_dom0)
+                cmd.extend((f"--{arg.replace('_', '-')}", str(value)))
+
+        if not cmd[2:]:
+            to_update = set()
+        else:
+            self.log.debug("Run command %s", " ".join(cmd))
+            output = subprocess.check_output(cmd)
+            self.log.debug("Command returns: %s", output.decode())
+
+            to_update = {
+                vm_name.strip() for vm_name
+                in output.decode().split("\n", maxsplit=1)[0]
```

Comment:
This will select just templates (the first line is `Following templates will be updated: ...`). But there may be also some standalones to update.